src/vendored/geoarrow/geoarrow.c

```diff
@@ -8972,7 +8972,7 @@ static int coords_wkt(struct GeoArrowVisitor* v, const struct GeoArrowCoordView*
 
   int64_t max_chars_needed = (n_coords * 2) +  // space + comma after coordinate
                              (n_coords * (n_dims - 1)) +  // spaces between ordinates
-                             ((private->precision + 1 + 5) * n_coords *
+                             ((private->precision + 1 + 10) * n_coords *
```

Comment:
This is just a random number for now to get my tests working (in spherely, still have to add some tests here). But so the main issue is that the number of 5 is too low for certain projected coordinates. How did you choose this number? For lon/lat in range of 0-180, this was always sufficient, but e.g. with Mercator you can easily get numbers in the millions. And I am not sure there is some theoretical max (the max float value would then be unrealistically big)

Reply:
Thanks!